packages/components/src/docs/Migration/ui-offcanvas-panel-mdx

```diff
@@ -0,0 +1,81 @@
+'# `ui-offcanvas-panel` to `sd-drawer` Migration Guide
+
+Solid Components has introduced the new `sd-drawer` component as a replacement for the `ui-offcanvas-panel`, with more intuitive slots for content organization and an improved API. This guide will help you migrate from the old component to the new one. Please refer to the `sd-drawer` documentation for a full overview of all attributes and events.
+
+## ✨ New Slots: `header` and `footer`
+
+The `sd-drawer` introduces new slots for `header` and `footer`, allowing you to structure your content more effectively. The `header` is typically used for a title, while the `footer` can contain buttons representing various options.
+
+```html
+<sd-drawer>
+  <div slot="header">Your Title Here</div>
+  <div slot="footer">
+    <button>Cancel</button>
+    <button>Confirm</button>
+  </div>
+  Your main content goes here...
+</sd-drawer>
+```
+
+## ❌ Removed Attributes: `darken`, `closeOutside` and `activeBelow`
+
+The `darken`, `closeOutside` and `activeBelow` attributes are no longer provided. The `darken` attribute previously controlled the appearance of the overlay, and `closeOutside` allowed users to close the drawer by clicking outside of it. The new `sd-drawer` design makes these features redundant.
+
+## 🔀 Updated Attribute: `position` to `placement`
+
+The `position` attribute has been replaced with `placement`. The `placement` attribute indicates the direction from which the drawer will open, with the options being 'end' or 'start'.
+
+```html
+<sd-drawer placement="start">
+  <!-- Your content -->
+</sd-drawer>
+```
+
+## ❌ Removed Event: `offcanvasTrigger`
+
+The `offcanvasTrigger` event has been removed. The open state of the drawer is now controlled through the `open` property or the `show()` and `hide()` methods.
+
+```html
+<sd-drawer open>...</sd-drawer>
+```
+
+or
+
+```
+const drawer = document.querySelector('sd-drawer');
+drawer.show(); // or drawer.hide();
+```
+
+## ❌ Removed Components: `ui-offcanvas-trigger` and `ui-offcanvas-provider`
+
+The `ui-offcanvas-trigger` and `ui-offcanvas-provider` components are no longer needed. The `sd-drawer` can be controlled directly through its properties and methods, simplifying its usage.
+
+## ✨ New Properties: `label` and `no-header`
+
+The `label` property allows you to provide an accessible name for the drawer, which is essential for proper accessibility.
+
+The `no-header` property lets you remove the default header, including the close button. If you use this property, make sure you provide an easy, accessible way for users to dismiss the drawer.
+
+```html
+<sd-drawer label="My Drawer" noHeader>
+  <!-- Your content -->
+</sd-drawer>
+```
+
+## ❌ Discouraged use of `contained`
+
+The `contained` property is a hidden feature meant only for testing purposes. It is discouraged to use it in production.
```

Comment:
I wouldn't even name it, as it's hidden everywhere in Storybook. People will do stuff. 😆

Reply:
Fair enough haha